### ISSUE TITLE
False alarm "Wrong number of type arguments" with const generics

### ISSUE THREAD

**Issue by filtsin** [intellij-rust/intellij-rust#5753]
## Environment

* **IntelliJ Rust plugin version:** 0.3.126.3220-201
* **Rust toolchain version:** rustc 1.46.0-nightly (5c9e5df3a 2020-07-16)
* **IDE name and version:** CLion 2020.1.2
* **Operating system:** Linux 5.6.15-arch1-1

## Problem description
Example of code:
```rust
#![feature(const_generics)]

struct Foo<T, const N: i32> {
  data: T
}

impl<T, const N: i32> Foo<T, N> {}

fn main() {}
```
On impl line we have error: `Wrong number of type arguments: expected 1, found 2 [E0107]`.

Also, the following error is quite annoying: `const generics is experimental [E0658]`. How can I turn it off? All code is red because of it.

**Comment by NyxCode** [comment]
Currently encountering the same error.
You can `alt+enter`, click on the inspection and select "disable".
